homeassistant/components/blebox/climate.py

```diff
@@ -81,9 +81,9 @@ def target_temperature(self):
     async def async_set_hvac_mode(self, hvac_mode):
         """Set the climate entity mode."""
         if hvac_mode == HVAC_MODE_HEAT:
-            return await self._feature.async_on()
+            await self._feature.async_on()
```

Comment:
We want to return after this, right?
```suggestion
            await self._feature.async_on()
            return
```

Reply:
Fixed in 74e43b4. (Seems I weirdly didn't rerun my local tests before pushing).

I also updated the tests to make the cause of failing tests more obvious.